src/Symfony/Component/HttpKernel/DataCollector/ConfigDataCollector.php

```diff
@@ -270,60 +270,20 @@ public function getName()
      *
      * @return string One of: unknown, dev, stable, eom, eol
      */
-    private function requestSymfonyState()
+    private function calculateSymfonyState()
```

Comment:
it doesn't really calculate. I think "determine" is a better word in this context? (not a native though...)